queue_job_group/static/src/xml/systray.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+
+    <t t-name="queue_job_group.view.Menu">
+        <li class="o_mail_navbar_item o_job_group_navbar_item o_hidden">
+            <a class="dropdown-toggle" data-toggle="dropdown"
+               aria-expanded="false" title="Attendance Warnings" href="#">
+                <i class="fa fa-list"/>
+                <span class="o_notification_counter badge"/>
+            </a>
+            <ul class="o_mail_navbar_dropdown dropdown-menu o_job_group_navbar_dropdown" role="menu">
+                <li class="o_mail_navbar_dropdown_top">
+                    <div>
+                        <button type="button" class="btn btn-sm o_filter_button active">Job groups</button>
+                    </div>
+                    <button type="button" class="btn btn-sm o_view_all_group_jobs"> View All </button>
```

Comment:
Maybe you could add this style to the view all button, I think it will fit more
```css
.o_view_all_group_jobs {
    .btn-link;
    padding: 5px;
}
```